catroid/src/main/java/org/catrobat/catroid/ui/controller/LookController.java

```diff
@@ -154,7 +155,10 @@ private void handleDetails(LookData lookData, LookViewHolder holder, LookBaseAda
 				holder.lookFileSizeTextView.setText(UtilFile.getSizeAsString(new File(lookData.getAbsolutePath())));
 			}
 			int[] measure = lookData.getMeasure();
-			String measureString = measure[0] + " x " + measure[1];
+			int Length = measure[0];
+			int Width = measure[1];
```

Comment:
This code is working correctly, but there is a mixup with the naming of local variables:
lookData.getMeasure() will return an array looking like this: {width, height}
So your 'Length' should be renamed to 'width' and 'Width' should be renamed to 'height'. (Those local variables should be written in camelCase, else our checkstyle will complain ;) )